packages/skills/carpark_client/dialogues.py

```diff
@@ -178,7 +179,8 @@ def _next_dialogue_id(self) -> int:
 
         :return: the next id
         """
-        self._dialogue_id += 1
+        self._dialogue_id = random.randint(0, 1000000)
```

Comment:
I propose we revert this. We will develop a dialogue error handler next week to address the issue properly.

Reply:
I don't think I agree.

I agree that we should have a "duplicate dialogue id" error handler. However, what do you envisage being the correct process to select a dialogue new id? 

I think the simplest (from a user perspective) is that each time you start a dialogue, you generate a unique id. Ideally this would be a GUID, but a random number will approximate this for the moment.

Then yes, we should have an error handler in case someone creates two dialogues with the same id (which is perhaps indistinguishable from some simply violating the message ordering protocol). The error handler should obviously be tested by the automated test routines (in case you are worried about a bug in it being hidden).

Reply:
The issue is we cannot use a GUID as it is an `int` field on the OEF SDK. We can eventually do the GUID like approach plus dialogue id error handler. The correct process is iteration of the id in my opinion as this gives a straightforward recovery when an error is detected > jump to the `dialogue_id` proposed by the opponent (i.e. the last one he knows from you). Note, the dialogue is not uniquely identified by the `dialogue_id`, for unique identification we have the `dialogue_label`!

Reply:
Note, Ethereum does something similar with the `nonce` of transactions. You increment the `nonce` for each new transaction.

Reply:
OK - if we are limited by the OEF implementation then I am persuaded by your argument.